README.md

```diff
@@ -90,9 +209,12 @@ func main() {
 
 ```
 
-### Sending data to Wavefront
+## Send Data to Wavefront
 
-Use the `Sender` interface for sending data to Wavefront.
+Wavefront supports different metric types, such as gauges, counters, delta counters, histograms, traces, and spans. See [Metrics](https://docs.wavefront.com/metric_types.html) for details. To send data to Wavefront using `Sender` you need to instantiate the following:
```

Comment:
Are we really instantiating anything here? Maybe: 

If you want to send data to Wavefront using `Sender`, you have to modify `Sender` for the type of data you want to send, as follows: